charts/tidb-cluster/templates/config/_drainer-config.tpl

```diff
@@ -83,14 +83,11 @@ port = {{ .Values.binlog.drainer.mysql.port | default 3306 }}
 #schema = "tidb_binlog"
 {{- end }}
 
-{{- if eq .Values.binlog.drainer.destDBType "pb" }}
-# Uncomment this if you want to use pb or sql as db-type.
-# Compress compresses output file, like pb and sql file. Now it supports "gzip" algorithm only.
-# Values can be "gzip". Leave it empty to disable compression.
+{{- if or (eq .Values.binlog.drainer.destDBType "pb") (eq .Values.binlog.drainer.destDBType "file") }}
+# Uncomment this if you want to use file as db-type.
 [syncer.to]
-# directory to save pb file, default same as data-dir(save checkpoint file) if this is not configured.
-dir = "/data/pb"
-compression = "gzip"
+# directory to save binlog file, default same as data-dir(save checkpoint file) if this is not configured.
+dir = "data.drainer"
```

Comment:
actually, this change is not backward compatible. i copy-pasted from the latest drainer config template but we can change it back to `/data/pb` if needed.

Reply:
oh yes, this should be `/data/pb` because the drainer Pod mount the PV to `/data`

Reply:
good call. reverted to `/data/pb`.